controllers/get-article-comments.controller.js

```diff
@@ -0,0 +1,13 @@
+const { selectArticleComments } = require("../models/topics.models")
+
+const getArticleComments = (req,res)=>{
+    const {article_id} = req.params
+    if(!Number(article_id)){
+        return res.status(400).send({msg:"Article ID must be a number"})
+    }
```

Comment:
This is not how we should be sending back errors. This should be done in the model and as a promise reject, which allows your .catch to deal with this by sending it to your middleware error handlers